# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/libcubeb/moz.yaml
# Commit: 9794b0a3bffc
# Full Hash: 9794b0a3bffc474951b33527b738488717caffbf
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2019-12-12 21:50:40
# Regressor Bug: 1603384
# File Overlap Count: 1
# Description:
#   Bug 1603384 - Update cubeb from upstream to aa63601. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D56872
# ==============================================================================

diff -r 7d77d727cd1e -r 9794b0a3bffc media/libcubeb/moz.yaml
--- a/media/libcubeb/moz.yaml	Thu Dec 12 17:46:46 2019 +0200
+++ b/media/libcubeb/moz.yaml	Thu Dec 12 15:24:23 2019 +0000
@@ -19,5 +19,5 @@
   license: "ISC"
 
   # update.sh will update this value
-  release: "a1200c3437419c4c8487df4a065782c7d1b7245f (2019-10-10 07:52:05 +1300)"
+  release: "aa636017aca8f79ebc95fb9f03b9333eaf9fd8fc (2019-12-12 12:13:07 +0200)"
 